src/test/resources/application.yml

```diff
@@ -0,0 +1,34 @@
+spring:
+  datasource:
+    url: jdbc:tc:postgresql:11.1:///integration-tests-db

```

Comment:
why do we use this url?

Reply:
This needs to be used to connect to the Postgres test container